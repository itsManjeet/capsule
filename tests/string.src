println ("Hello World");
println ("Hello " + "World");
println ("Hello" == "hello");
println ("Hello" == "Hello");
println ("Hello" != "hello");
println ("Hello" != "Hello");
println ("Hello" > "world");
println ("Hello" < "world");
println ("Hello" > "hey");
println ("Hello" < "hey");