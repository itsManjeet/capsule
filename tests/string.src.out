Hello World
Hello World
false
true
true
false
false
false
true
false
